main/static/src/script/common/util.coffee

```diff
@@ -43,7 +43,7 @@ window.init_time = ->
         else
           $(this).text date.fromNow()
         $(this).attr 'title', date.local().format 'dddd, MMMM Do YYYY, HH:mm:ss Z'
-      setTimeout arguments.callee, 1000 * 45
+      setTimeout temp_func, 1000 * 45
```

Comment:
other ideas are welcome: https://stackoverflow.com/a/12050920

Reply:
I'm no Javascript expert, but could the "temp_func" in this case read something like "temp_window_init_time" or "window_init_time_func" to make it more readable?

Reply:
That actually doesn't work :/

Reply:
should be OK now!